src/labs/labs-slice.ts

```diff
@@ -0,0 +1,42 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import Lab from '../model/Lab'
+import { AppThunk } from '../store'
+import LabRepository from '../clients/db/LabRepository'
+
+interface LabsState {
+  isLoading: boolean
+  labs: Lab[]
```

Comment:
I think that this should be added as part of the patient slice. It seems like this functionality is part of the patient module, so therefore it makes sense to have the patient slice own this state update. @fox1t any thoughts here?

I imagine the patient slice getting a new state field called `labs`, and then a thunk function called `findLabsPatientId` and the corresponding reducers.

Reply:
in that case, the Labs should be stored using the src/patients/patient-slice.ts:73?
instead of LabsRepository?